activation/atxdb.go

```diff
@@ -52,11 +52,10 @@ func (db *ActivationDb) ProcessAtx(atx *types.ActivationTx) {
 	db.log.Info("processing atx id %v, pub-epoch %v node: %v layer %v", atx.ShortId(), epoch, atx.NodeId.Key[:5], atx.PubLayerIdx)
 	err := db.ContextuallyValidateAtx(atx)
 	if err != nil {
-		db.log.Warning("ATX %v failed validation: %v", atx.ShortId(), err)
+		db.log.Error("ATX %v failed contextual validation: %v", atx.ShortId(), err)
```

Comment:
shouldn't we return here if atx is contextually invalid?

Reply:
Add Unit tests to test this scenario and make sure invalid atx is not stored in db

Reply:
no, a contextually invalid ATX **should** be stored in the DB. This is why we don't `return` and why we don't have a UT for this (it should only log at this time, later we'll want to block this miner -- talk to @zalmen if you disagree).